packages/ui-patterns/src/McpUrlBuilder/components/McpConfigurationDisplay.tsx

```diff
@@ -65,6 +65,7 @@ export function McpConfigurationDisplay({
         value={JSON.stringify(clientConfig, null, 2)}
         language="json"
         className="max-h-64 overflow-y-auto"
+        focusable={false}
```

Comment:
This in turn disables `contentEditable` which stops system spellcheck from showing red underlines